src/libudev/meson.build

```diff
@@ -43,7 +43,6 @@ libudev_pc = custom_target(
 ############################################################
 
 tests += [
-        [files('test-libudev.c'),
-         [libshared,
-          libudev_basic]],
+        { 'sources' : files('test-libudev.c'),
+          'link_with' : [libshared, libudev_basic] },
```

Comment:
Separate lines please.

Reply:
for link_with and dependencies? That's gonna be verbose…